app_specific_files/dummy_app_multicast/scripts/generate_random_files.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+: '
+    ** Copyright (c) 2019, Autonomous Networks Research Group. All rights reserved.
+    **     contributor: Quynh Nguyen, Bhaskar Krishnamachari
+    **     Read license file in main directory for more details
+'
```

Comment:
Is this an example for the public to learn from? If so, we need to create an issue of documenting this. Not something to do right now, but we can keep it in the books.

Reply:
Yes, I have to put description about this tool [https://github.com/ANRGUSC/Automatic-DAG-Generator](https://github.com/ANRGUSC/Automatic-DAG-Generator) on this part of documentation [https://jupiter.readthedocs.io/en/latest/Jexample.html#](https://jupiter.readthedocs.io/en/latest/Jexample.html#)

Reply:
Why is it a separate repository?

Reply:
Since it is a separate tool that you can use independently.